release.toml

```diff
@@ -0,0 +1,16 @@
+# we don't have a crate yet, so
+disable-publish = true
+# change when we all have gpg keys
+sign-commit = false
+dev-version-ext = 'dev'
+# update all crates in plume at once:
+consolidate-commits = true
+tag-prefix = ''
+
+pre-release-replacements = [
+  {file="CHANGELOG.md", search="Unreleased", replace="{{version}}"},
+  {file="CHANGELOG.md", search="\\.\\.\\.dev", replace="...{{tag_name}}", exactly=1},
+  {file="CHANGELOG.md", search="ReleaseDate", replace="{{date}}"},
+  {file="CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n\n## [Unreleased] - ReleaseDate", exactly=1},
+  {file="CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[Unreleased]: https://github.com/assert-rs/predicates-rs/compare/{{tag_name}}...HEAD", exactly=1},
```

Comment:
Shouldn't the repo URL be changed here?

Reply:
how?

Reply:
To something like `https://github.com/Plume-org/Plume/compare/{{tag_name}}...HEAD`, so that the changelog links to our commit history.

Reply:
Also, is there a `<!-- next-url -->` somewhere in the changelog you added? I can't find it.

Reply:
i didn't add it to our current changelog